### PR TITLE
Implement hex.offset config variable to hide address column from hex-dump

### LINKED ISSUE #13849: Implement "hex.address" config variable to hide address column from hexdump
**Is your feature request related to a problem? Please describe.**
Currently we have options to hide the Address\Offset column in Disassembly and Graph using `asm.offset` and `graph.offset`. There is no such opition for `hex.*` variables though `asm.offset` also removes the Address column from hexdump (`px`). 

**Describe the solution you'd like**
I think we should separate the two  and create another config variable named "hex.address".

**Additional context**
As for @trufae request, it'd be best to change `asm.offset` and `graph.offset` to `asm.address` and `graph.address`

### PR DESCRIPTION
**Pull request by kuqadk3**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

Implement hex.offset config variable to hide address column from hex-dump in radare2 (x, px)
...

**Test plan**

...

**Closing issues**

close #13849 
...

### CODE REVIEW

**Comment by kuqadk3** [comment]
Just change asm.offset to hex.offset for all px*.

**Comment by radare** [comment]
let's wait for travis. and ill merge

File: libr/core/cconfig.c [hunk 1]
```diff
@@ -3045,6 +3045,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETBPREF ("asm.middle", "false", "Allow disassembling jumps in the middle of an instruction");
 	SETBPREF ("asm.noisy", "true", "Show comments considered noisy but possibly useful");
 	SETBPREF ("asm.offset", "true", "Show offsets at disassembly");
+	SETBPREF ("hex.offset", "true", "Show offsets at hex-dump");
```
Comment (radare) [inline libr/core/cconfig.c]: imho it's "in the" instead of "at", same for asm.offset description

File: libr/core/cmd_print.c [hunk 2]
```diff
@@ -6079,7 +6079,7 @@ l = use_blocksize;
 		break;
 	case 'x': // "px"
 	{
-		int show_offset = r_config_get_i (core->config, "asm.offset");
+		int show_offset = r_config_get_i (core->config, "hex.offset");
```
Comment (radare) [inline libr/core/cmd_print.c]: this is bool

File: libr/core/cmd_print.c [hunk 3]
```diff
@@ -6207,8 +6207,9 @@ l = use_blocksize;
 			break;
 		case 'i': // "pxi"
 			if (l != 0) {
+				int show_offset = r_config_get_i (core->config, "hex.offset");
```
Comment (radare) [inline libr/core/cmd_print.c]: bool

File: libr/include/r_util/r_print.h [hunk 4]
```diff
@@ -146,7 +146,7 @@ R_API void r_print_unset_flags(RPrint *p, int flags);
 R_API void r_print_addr(RPrint *p, ut64 addr);
 R_API void r_print_section(RPrint *p, ut64 at);
 R_API void r_print_columns (RPrint *p, const ut8 *buf, int len, int height);
-R_API void r_print_hexii(RPrint *p, ut64 addr, const ut8 *buf, int len, int step);
+R_API void r_print_hexii(RPrint *p, ut64 addr, const ut8 *buf, int len, int step, int show_offset);
```
Comment (radare) [inline libr/include/r_util/r_print.h]: s/int/bool

File: libr/util/print.c [hunk 5]
```diff
@@ -755,7 +755,9 @@ R_API void r_print_hexii(RPrint *rp, ut64 addr, const ut8 *buf, int len, int ste
 		if (isAllZeros (buf + i, inc)) {
 			continue;
 		}
-		p ("%8X:", addr + i);
+		if (show_offset){
```
Comment (radare) [inline libr/util/print.c]: missing space before {

File: libr/util/print.c [hunk 6]
```diff
@@ -733,7 +733,7 @@ static bool isAllZeros(const ut8 *buf, int len) {
 }
 
 #define Pal(x,y) (x->cons && x->cons->context->pal.y)? x->cons->context->pal.y
-R_API void r_print_hexii(RPrint *rp, ut64 addr, const ut8 *buf, int len, int step) {
+R_API void r_print_hexii(RPrint *rp, ut64 addr, const ut8 *buf, int len, int step, int show_offset) {
```
Comment (radare) [inline libr/util/print.c]: this is bool too

Comment (radare) [reply libr/util/print.c]: but.. what about adding a field inside RPrint->show_offset instead of adding a parameter?

Comment (kuqadk3) [reply libr/util/print.c]: Yeah that sounds better, fixing it

**Comment by radare** [comment]
omg the r2r binary have been merged into master /o\ missed the blob in the review.

**Comment by ret2libc** [comment]
@radare are you going to remove it and add to gitignore?

**Comment by kuqadk3** [comment]
oh rip...sorry

**Pull request merged**